README.md

```diff
@@ -31,7 +31,7 @@ const options = {
 const AfricasTalking = require('africastalking')(options);
 
 // Initialize a service e.g. SMS
-sms = africastalking.SMS
+let sms = africastalking.SMS
```

Comment:
I might not say it won't work without `let` but the example will fail because 

1. Having defined `const options` twice
2. ReferenceError: `africastalking` is not defined

@wikiyodo, you can take that into consideration and thanks for noting it down.